### ISSUE TITLE
bug: curl missing from latest release of minio/mc, causes errors starting.

### ISSUE THREAD

**Issue by Malte0621** [revoltchat/self-hosted#69]
### What happened?

When pulling and starting the self-hosted version of revolt using docker, the `minio/mc` image spams **errors** that says "curl" is missing. Which is because **curl was removed** from the latest version of `minio/mc`.

*Temporary solution:*
Downgrade `minio/mc` to `RELEASE.2023-10-24T21-42-22Z`.
(This is done by changing the `docker-compose.yml` file like following:
```yml
createbuckets:
    image: minio/mc:RELEASE.2023-10-14T01-57-03Z
```
(note the added `:RELEASE.2023-10-14T01-57-03Z`)

***Permanent solution:***
* Get curl installed somehow?
* Don't use curl?

**Comment by Malte0621** [comment]
> Temporary fix is to change docker-compose.yml to use an older version of minio/mc from 5 months ago when it had curl still
> 
> ```diff
> + createbuckets:
> +    image: minio/mc:RELEASE.2023-07-21T20-44-27Z
> -  createbuckets:
> -    image: minio/mc
> ```

I already posted the fix lol (`minio/mc:RELEASE.2023-10-14T01-57-03Z` works as it has curl .-.)

**Comment by Geoxor** [comment]
> > Temporary fix is to change docker-compose.yml to use an older version of minio/mc from 5 months ago when it had curl still
> > ```diff
> > + createbuckets:
> > +    image: minio/mc:RELEASE.2023-07-21T20-44-27Z
> > -  createbuckets:
> > -    image: minio/mc
> > ```
> 
> I already posted a temporary fix lol (`minio/mc:RELEASE.2023-10-14T01-57-03Z` works as it has curl .-.)

Yeah sorry i didn't see it and figured a fix myself and then noticed you mentioned it

**Comment by insertish** [comment]
Fixed in #80
